x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2652,6 +2652,52 @@ export class RulesClient {
     }
   }
 
+  public async runSoon({ id }: { id: string }): Promise<void> {
+    return await retryIfConflicts(
+      this.logger,
+      `rulesClient.runSoon('${id}')`,
+      async () => await this.runSoonWithOCC({ id })
```

Comment:
Usually we use OCC when we're updating the rule saved object and may run into conflicts that we want to retry, but I don't think this is the case here. I think we can absorb the helper `runSoonWithOCC` function

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/40b50cb5653c2e5937115cea9bc2858f4f05cf7a